save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectInfoMenu.kt

```diff
@@ -0,0 +1,227 @@
+            url = "$apiUrl/contests/${props.organizationName}/${props.projectName}/best",
+            headers = Headers().also {
+                it.set("Accept", "application/json")
+            },
+            loadingHandler = ::loadingHandler,
+        )
+            .unsafeMap {
+                it.decodeFromJsonString()
+            }
+        setBestResults(results)
+    }()
+
+    val (project, setProject) = useState(Project.stub(-1))
+    useRequest(isDeferred = false) {
+        val projectFromBackend: Project = get(
+            url = "$apiUrl/projects/get/organization-name?name=${props.projectName}&organizationName=${props.organizationName}",
+            headers = Headers().also {
+                it.set("Accept", "application/json")
+            },
+            loadingHandler = ::loadingHandler,
+        )
+            .unsafeMap {
+                it.decodeFromJsonString()
+            }
+        setProject(projectFromBackend)
+    }()
+
+    div {
+        className = ClassName("d-flex justify-content-between")
+        div {
+            className = ClassName("col-3")
+            div {
+                className = ClassName("text-xs text-center font-weight-bold text-primary text-uppercase mb-3")
+                +"Best contest scores"
+            }
+            ul {
+                className = ClassName("list-group")
+                bestResults.forEach {
+                    li {
+                        className = ClassName("list-group-item pl-0 pr-0 pb-0 pt-0")
+                        a {
+                            href = "#/contests/${it.contestName}"
+                            className = ClassName("stretched-link")
+                        }
+                        contestResultCard {
+                            name = it.contestName
+                            contestScore = it.score.toDouble()
+                        }
+                    }
+                }
+            }
+        }
+
+        div {
+            className = ClassName("col-3")
+            div {
+                className = ClassName("text-xs text-center font-weight-bold text-primary text-uppercase mb-3")
+                +"Members"
+            }
+            div {
+                className = ClassName("latest-photos")
+                div {
+                    className = ClassName("row")
+                    usersInProject.forEach { user ->
+                        div {
+                            className = ClassName("col-md-4")
+                            figure {
+                                img {
+                                    className = ClassName("img-fluid")
+                                    src = user.avatar?.let { path ->
+                                        "/api/$v1/avatar$path"
+                                    }
+                                        ?: run {
+                                            "img/user.svg"
+                                        }
```

Comment:
We must've already added similar logic for avatars somewhere. Can it be extracted into a reusable function?

Reply:
Yes, will do it